teamwork/templates/base.html

```diff
@@ -1,23 +1,78 @@
-    <link rel="stylesheet" href="{% static 'css/base.css' %}">
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-  </body>
-</html>
+      <!-- Custom CSS -->
+      <link rel="stylesheet" href="{% static 'css/main.css' %}">
+    </head>
+    <body>
+      <!-- Navigation bar -->
+      <nav class="navbar navbar-inverse navbar-static-top" role="navigation">
+                      <div class="container">
+                          <div class="navbar-header">
+                              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+                                  <span class="sr-only">Toggle navigation</span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                                  <span class="icon-bar"></span>
+                              </button>
+                              <p class="navbar-brand">{{ project.name }}</p>
+                          </div>
+                          <div class="navbar-collapse collapse">
+                              <ul class="nav navbar-nav">
+                                  <li class="active"><a href="{% url 'dashboard' user.username %}">Dashboard</a>
+                                  </li>
+                                  <li><a href="calendar">Calendar</a>
+                                  </li>
+                                  <li><button type="button" class="btn btn btn-default btn-success navbar-btn"> <span class="glyphicon glyphicon-plus"></span> Add new project </button>
+                                  </li>
+                              </ul>
+
+                              <ul class="nav navbar-nav navbar-right">
+                                <li class="dropdown">
+                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown"><span class="glyphicon glyphicon-user"></span> {{ user.username }} <b class="caret"></b></a>
+                                    <ul class="dropdown-menu">
+                                        <li><a href="#"><span class="glyphicon glyphicon-cog"></span> Settings</a>
+                                        </li>
+                                        <li class="divider"></li>
+                                        <li><a href="#">Log out</a>
+                                        </li>
+                                    </ul>
+                                </li>
+                              </ul>
+                          </div>
+                          <!--/.nav-collapse -->
+                      </div>
+        </nav>
+      {% block content %}
+      {% endblock %}
+
+      <!--Footer -->
+      <footer class="container-fluid text-center">
+        <p><a href="https://mycourses.aalto.fi/" title="MyCourse" target="_blank">My Course</a>  |  All rights reserved</p>

```

Comment:
1st comment: With `target="_blank"`, the user isn't in control and can only open the link in a new tab. Without it the user can either open the link normally or right click and select "open in a new tab" if they want to. Also, `target="_blank"` can be disorienting for people who use screen readers or have low vision and can't properly see the tabs. I leave the choice to you but if you want to use `target="_blank"` you need to add `rel="noopener noreferrer"` in the `<a>` tag to fix the security issue.

3rd comment: I think we shouldn't put in any legal text that we don't mean. I'm fine with 'Created by...' or something like that if the footer visually needs something extra to fill it.